### PR TITLE
Bug/temp

### PR DESCRIPTION
**Pull request by wentianyang66**

### CODE REVIEW

File: .bin/acorn [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```
Comment (brucexu-eth) [inline .bin/acorn:1]: seem irrelevant, add to gitignore?

Comment (wentianyang66) [reply .bin/acorn]: deleted
